packages/turf-meta/test.ts

```diff
@@ -659,10 +682,10 @@ const geojsonSegments = featureCollection([
 ]);
 
 test("segmentEach -- index & subIndex", (t) => {
-  const featureIndexes = [];
-  const multiFeatureIndexes = [];
-  const geometryIndexes = [];
-  const segmentIndexes = [];
+  const featureIndexes: (number | undefined)[] = [];
+  const multiFeatureIndexes: (number | undefined)[] = [];
+  const geometryIndexes: (number | undefined)[] = [];
+  const segmentIndexes: (number | undefined)[] = [];
```

Comment:
There's a bunch of these functions in meta that pass optional *Index values to the callback, when I think they're always specified and not optional. We need to do a pass and clean that up so this sort of thing can be `number[]` as much as possible.